sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -67,6 +67,24 @@ object functions {
   def column(colName: String): Column = Column(colName)
 
   /**
+   * Convert a number from one base to another for the specified expressions
+   *
+   * @group math_funcs
+   * @since 1.5.0
+   */
+  def conv(num: Column, fromBase: Column, toBase: Column): Column =
+    Conv(num.expr, fromBase.expr, toBase.expr )
+
+  /**
+   * Convert a number from one base to another for the specified expressions
+   *
+   * @group math_funcs
+   * @since 1.5.0
+   */
+  def conv(num: String, fromBase: String, toBase: String): Column =
```

Comment:
Probably more user cases of `fromBase` and the `toBase` will be represented as the real value, than the column names? What do you think? 
@rxin we actually have some cases like this, do you think if we need to support both constant string values and column names for cases like this?